QKMRZScanner/QKMRZScannerView.swift

```diff
@@ -29,6 +29,9 @@ public class QKMRZScannerView: UIView {
     @objc public dynamic var isScanning = false
     public weak var delegate: QKMRZScannerViewDelegate?
     
+    /// If true device will vibrate on a successful scan
+    public var hapticsOnSuccess: Bool = true
```

Comment:
Could you please remove the explicitly defined data type (`Bool`) as well as the comment above? I might add a proper documentation for every public property and method all together at some point later.

*I would then also place the var just below the `isScanning` property :)